connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -225,6 +225,7 @@ protected static ConfigDef baseConfigDef() {
                 .define(PLUGIN_PATH_CONFIG,
                         Type.LIST,
                         null,
+                        ConfigDef.ValidList.anyNonDuplicateValues(false, true),
```

Comment:
quoted from  [KIP-898](https://cwiki.apache.org/confluence/display/KAFKA/KIP-898%3A+Modernize+Connect+plugin+discovery)

> This will be equivalent to extracting the plugin.path  configuration from the worker properties and specifying --plugin-path .

Perhaps we should add manual check for the tool. 

https://github.com/apache/kafka/blob/7426629ba4e5f4e3feb177e3987d54c1c078ff17/tools/src/main/java/org/apache/kafka/tools/ConnectPluginPath.java#L173

Reply:
@majialoong Could you please file a minor patch for it?